Vector.sc

```diff
@@ -1,92 +1,106 @@
-// �2009 Miguel Negr�o, Fredrik Olofsson
+// 2009 Miguel Negro, Fredrik Olofsson
 // GPLv3 - http://www.gnu.org/licenses/gpl-3.0.html
 
 //Vector class for vectors of any dimension, with optimized classes for 2D and 3D.
 
 AbstractVector[slot] : ArrayedCollection {
-	
+
 	species { ^this.class }
-	
+
 	//usual notation up to 3D vectors
 	x { ^this.at(0) }
-	
+
 	y { ^this.at(1) }
-	
-	z { ^this.at(2) }	
-	
+
+	z { ^this.at(2) }
+
 	//inner product
-	<|> { |vec| 
-		^this.subclassResponsibility 
+	<|> { |vec|
+		^this.subclassResponsibility
 	}
-	
+
 	norm {
 		^(this <|> this).sqrt
 	}
-	
+
 	dist { |vec|
 		^(this - vec).norm
 	}
-	
+
 	normalize {
 		^this / this.norm
 	}
-	
+
 	limit { |max|
 		if(this.norm > max, { ^this.normalize * max })
 	}
-	
+
```

Comment:
Indentation seems wrong here. Possibly not using tabs ?

Reply:
This is caused by the removal of trailing whitespace. Same thing happens all over the file.